vscode-ext/src/extension.ts

```diff
@@ -78,6 +78,7 @@ class MirrordAPI {
 				throw new Error("Unsupported platform: " + process.platform + " " + process.arch);
 			} else {
 				console.log("Running in WSL.");
```

Comment:
This is not from this PR, but shouldn't this be 
```suggestion
				console.log("Running in Windows.");
```
?